clients/src/main/java/org/oracle/okafka/common/utils/FetchOffsets.java

```diff
@@ -0,0 +1,311 @@
+package org.oracle.okafka.common.utils;
+
+import java.sql.CallableStatement;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.sql.Types;
+
+import org.oracle.okafka.common.requests.ListOffsetsResponse.ListOffsetsPartitionResponse;
+
+public class FetchOffsets {
+
+	public static ListOffsetsPartitionResponse fetchEarliestOffset(String topic, int partition, Connection jdbcConn)
+			throws SQLException {
+		ListOffsetsPartitionResponse response = new ListOffsetsPartitionResponse().setPartitionIndex(partition);
+		CallableStatement cStmt = null;
+		String plsql = """
```

Comment:
Have class level static variable for this queries.

Reply:
okay